kafka/src/main/java/org/apache/apex/malhar/kafka/AbstractKafkaInputOperator.java

```diff
@@ -194,6 +194,11 @@ public void deactivate()
   }
 
   @Override
+  public void beforeCheckpoint(long windowId)
```

Comment:
For some reason I am not seeing this change when I merge this pull request into my local branch. @tweise do you have a fresh merge from this pull request?